updater/lib/dependabot/updater/operations/create_group_security_update_pull_request.rb

```diff
@@ -15,6 +15,7 @@ class CreateGroupSecurityUpdatePullRequest
         include GroupUpdateCreation
 
         def self.applies_to?(job:)
+          return false if Dependabot::Experiments.enabled?(:grouped_security_updates_disabled)
```

Comment:
I recognize that it's a bit weird to phrase it like this, but I need the default case to be grouped_security_updates being on and the way Dependabot::Experiments works is it's falsy by default.